templates/problem/editorial.html

```diff
@@ -37,9 +37,12 @@
             {% endif %}
         {% endwith %}
         {{ solution.content|markdown('solution', MATH_ENGINE)|reference|str|safe }}
-    </div>
-    <hr>
-    {% include "comments/list.html" %}
+    </div> 
+       
+    {% if not disable_comments %}
```

Comment:
The comments used to be inside the `<div/>`, but are now outside. Was this change intentional?

Reply:
If I'm not wrong, is outside the "div" in the current main branch from where I branched:
https://github.com/DMOJ/online-judge/blob/master/templates/problem/editorial.html